libraries/AP_InertialSensor/AP_InertialSensor_Backend.h

```diff
@@ -53,6 +53,15 @@ class AP_InertialSensor_Backend
      */
     virtual bool update() = 0; /* front end */
 
+    // if AP_INERTIALSENSOR_FAST_SAMPLE_WINDOW_ENABLED
```

Comment:
Yes, because of the vehicle include protections it can't actually be defined here, the comment is just to indicate that the code is technically controlled